web-bundle/src/main/java/com/graphhopper/http/GraphHopperManaged.java

```diff
@@ -122,6 +123,15 @@ public GraphHopperManaged(GraphHopperConfig configuration, ObjectMapper objectMa
         }
         configuration.setProfiles(newProfiles);
 
+        // Error if gtfs_link_mapper profile wasn't included in GH config (link mapper step will fail in this case)
+        if (configuration.has("gtfs.file")) {
+            if (newProfiles.stream()
+                    .filter(p -> p.getName().equals("gtfs_link_mapper"))
+                    .collect(Collectors.toSet()).size() == 0) {
+                throw new RuntimeException("Graphhopper config must include gtfs_link_mapper profile!");
```

Comment:
should we also check that it's present in the contraction hierarchy config so we don't get super slow link mapping if someone mistakenly deletes it?

Reply:
Hmm, good though, that seems like a good idea - from my "testing" (where I messed this up), it seems like the link mapper ran without loudly failing when I didn't include the contraction hierarchy profile declaration, but no routes were actually found (it wasn't just super slow, something was actually broken). Definitely want to prevent that situation in the future.

I'll try to see if this is possible!